listings/applications/erc721/src/erc721.cairo

```diff
@@ -0,0 +1,210 @@
+use starknet::ContractAddress;
+
+#[starknet::interface]
+pub trait IERC721<TContractState> {
```

Comment:
The `IERC721` interface in the PR does not match the conventional one
     - `IERC721` does not contain functions to get `(get_)name`, `(get_)symbol` and `(get_)token_uri`, those are part of `IERC721Metadata`.
     - The `IERC721Metadata` interface is missing
     - You can use OpenZeppelin's ERC721 implementation as inspiration for this PR, it will make it much easier to implement, see https://github.com/OpenZeppelin/cairo-contracts/blob/main/src/token/erc721

Reply:
Yes, a simplified version of OpenZeppelin is really enough for this!